discovery-infra/tests/test_e2e_install.py

```diff
@@ -1,32 +1,40 @@
 import time
-from typing import Tuple
 
 import pytest
+from _pytest.fixtures import FixtureLookupError
 from junit_report import JunitTestSuite
+
 from test_infra.consts import OperatorStatus
 
 from tests.base_test import BaseTest
-from tests.config import ClusterConfig, TerraformConfig
+from tests.config import ClusterConfig
 from tests.conftest import get_available_openshift_versions, get_api_client
 
 
 class TestInstall(BaseTest):
 
+    @pytest.fixture
+    def cluster_configuration(self, request):
+        config = ClusterConfig()
+
+        try:
+            version = request.getfixturevalue("openshift_version")
+            config.openshift_version = version
+        except FixtureLookupError:
+            pass
+
+        return config
+
     @JunitTestSuite()
     @pytest.mark.parametrize("openshift_version", get_available_openshift_versions())
-    def test_install(self, configs: Tuple[ClusterConfig, TerraformConfig], get_nodes, get_cluster, openshift_version):
-        cluster_config, tf_config = configs
-        cluster_config.openshift_version = openshift_version
-        new_cluster = get_cluster(get_nodes(tf_config, cluster_config), cluster_config)
-        new_cluster.prepare_for_installation()
-        new_cluster.start_install_and_wait_for_installed()
+    def test_install(self, cluster, openshift_version):
+        cluster.prepare_for_installation()
+        cluster.start_install_and_wait_for_installed()
 
     @JunitTestSuite()
     @pytest.mark.parametrize("sleep_time", [1, 60])
-    def test_dummy(self, configs: Tuple[ClusterConfig, TerraformConfig], get_nodes, get_cluster, sleep_time):
-        cluster_config, tf_config = configs
-        new_cluster = get_cluster(get_nodes(tf_config, cluster_config), cluster_config)
-        new_cluster.prepare_for_installation()
+    def test_dummy(self, cluster, sleep_time):
+        cluster.prepare_for_installation()
         time.sleep(sleep_time)
```

Comment:
What's this test? @eliorerz :)

Reply:
irrelevant,  please open a new task for this test...